pkg/cmd/webapp/init/init.go

```diff
@@ -161,32 +161,81 @@ func (cmd *InitCmd) run(info *InitInfo, options *contracts.AzionApplicationOptio
 		}
 	}
 
+	var projectName, projectSettings string
 	if shouldFetchTemplates {
-		if err := cmd.fetchTemplates(info); err != nil {
+		if err = cmd.fetchTemplates(info); err != nil {
 			return err
 		}
 
-		if err = UpdateScript(info, cmd, path); err != nil {
+		bytePackageJson, pathPackageJson, err := ReadPackageJson(cmd, path)
+		if err != nil {
+			return err
+		}
+
+		projectName, projectSettings, err = DetectedProjectJS(bytePackageJson)
```

Comment:
if we don't detect the framework, we should check the flag --type to see if the user has informed any.